DokanNet/IDokanOperations.cs

```diff
@@ -381,4 +404,11 @@ public interface IDokanOperations
         /// \see <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/aa365993(v=vs.85).aspx">About KTM (MSDN)</a>
         NtStatus FindStreams(string fileName, out IList<FileInformation> streams, DokanFileInfo info);
     }
+}
+
+/// <summary>
+/// Namespace for AssemblyInfo and resource strings
+/// </summary>
+namespace DokanNet.Properties
```

Comment:
I have clarified purpose of the blank namespace with at code comment so that it can not accidentally be removed in the future.